content/applications/websites/website/web_design/building_blocks/structure.rst

```diff
@@ -0,0 +1,69 @@
+================
+Structure blocks
+================
+
+The website configurator provides a range of :guilabel:`Structure` :doc:`building blocks <../building_blocks>`
+to design your website's layout, including elements like headings, images, and text.
+
+To add a block to your web page, click :guilabel:`Edit`, select a block from the
+:guilabel:`Structure` section, and drag and drop it to your website.
+
+.. image:: structure/structure-blocks.png
+   :alt: Drag and drop a block from the structure section.
+   :scale: 75%
```

Comment:
Why are you scaling all your images?

Reply:
It's a tip I got from Loredana, so that we don't lose quality and the images don't take up too much space on screen.

Reply:
I would suggest using fewer screenshots, reducing their resolution, or hiding them behind a spoiler box (a new idea I got recently)